tests/specs/framework-detection.e2e.js

```diff
@@ -0,0 +1,61 @@
+import { supportsFetchExtended } from '../../tools/browser-matcher/common-matchers.mjs'
+import { getMetricsFromResponse } from './helpers/assertion-helpers'
+
+const config = {
+  init: {
+    page_view_timing: {
+      enabled: false
+    }
+  }
+}
+
+describe('framework detection', () => {
+  withBrowsersMatching(supportsFetchExtended)('detects a page built with REACT and sends a supportability metric', async () => {
+    const rumPromise = browser.testHandle.expectRum()
+    await browser.url(await browser.testHandle.assetURL('frameworks/react/simple-app/index.html', config)).then(() => browser.waitForAgentLoad())
+
+    await rumPromise.then(async () => {
+      browser.url(await browser.testHandle.assetURL('/'))
+      return browser.testHandle.expectMetrics(3000)
+    }).then((response) => {
```

Comment:
Would rather use `async..await` instead of promise chains. 

```suggestion
    const [unloadSupportMetricsResults] = Promise.all([
      browser.testHandle.expectMetrics(3000),
      browser.url(await browser.testHandle.assetURL('/'))
    ])
```